set self.colors to a copy of the colormap list

slabasan opened the pull request:
Original assignment of colormaps to self.colors resulted in both lists pointing
to the same list. When reversing self.colors, this impacted the colormap and
colors list, resulting in inconsistent tree outputs when making multiple
calls to tree().

slabasan commented:
This is passing, but the report doesn't appear to show up on codecov.io. I see a token to use when uploading reports, but I'm not sure where to apply this.

tgamblin commented:
We weren't actually running the coverage reports 😄.  But after adding a `coverage run` to the script, we're [in business](https://codecov.io/gh/llnl/hatchet).  Now, unfortunately, we're only 54% covered -- we'll have to work on that!

slabasan committed:
set self.colors to a copy of the colormap list

Original assignment of colormaps to self.colors resulted in both lists pointing
to the same list. When reversing self.colors, this impacted the colormap and
colors list, resulting in inconsistent tree outputs when making multiple
calls to tree().
